app/src/main/java/org/listenbrainz/android/viewmodel/SocialViewModel.kt

```diff
@@ -164,4 +170,12 @@ class SocialViewModel @Inject constructor(
             }
         }
     }
+
+   fun getFollowers(username : String) : Resource<SocialData> = runBlocking {
```

Comment:
This is wrong. This will freeze UI because of the await() call. This is because android relies on actual parallelism (mostly) instead of scheduling(Still happens). make this function suspend to make UI aware that this is a synchronous function and UI should decide how it wants to call this function. Usually the UI would call this using lifecycleScope or LaunchedEffect and store it in a state.